src/clj/clojurians_log/time_util.clj

```diff
@@ -51,3 +52,22 @@
 (defn html-ts->time
   [ts]
   (jt/zoned-date-time DateTimeFormatter/RFC_1123_DATE_TIME ts))
+
+(defn day-interval
+  [y m d]
+
```

Comment:
When there's no docstring than place the argument vector on the same line as the function name.